cython_lint/cython_lint.py

```diff
@@ -849,9 +852,19 @@ def main(argv: Sequence[str] | None = None) -> int:  # pragma: no cover
     parser.add_argument('--max-line-length', type=int, default=88)
     parser.add_argument('--no-pycodestyle', action='store_true')
     parser.add_argument('--version', action='version', version=__version__)
+    parser.add_argument(
+        '--ignore',
+        type=str,
```

Comment:
I think `nargs='*'` would work (then there should be no need for the `if-then` below)

Reply:
Not sure how ``nargs='*'`` would help here. If empty it would still return `None`. I used ``default`` instead